data/ka'het/ka'het missions.txt

```diff
@@ -293,3 +293,409 @@ mission "Ringworld Debris: Quarg"
 		log "Asked the Quarg about the history of the Graveyard. Was told that the ringworld the Quarg had there was lost in their war with the Pug, and have learned about the previous inhabitants of the region, the Builders."
 		log "Factions" "Builders" `The Builders inhabited the Graveyard millennia ago, but have since become extinct. They developed some sort of weapon, called the "Ka'het," which stopped obeying their orders and rebelled against them.`
 		log "Factions" "Ka'het" `According to the Quarg, the Ka'het were initially a weapon created by the Builders, which then rebelled and led them to extinction. Even in the millennia that passed, they are still being produced in the Graveyard.`
+
+mission "Ka'het: Patir Reveal"
+	invisible
+	landing
+	to offer
+		has "Remnant: Cognizance 19: done"
+	on offer
+		event "patir appears" 3
+
+event "patir appears"
+	system Patir
+		remove hidden
+
+mission "Ka'het: Patir Mystery 0"
+	invisible
+	landing
+	on enter "Patir"
+		dialog
+			`As the jumping sequence comes to an end, you find yourself orbiting a small black hole, surrounded by nothing but countless asteroids and debris. Even more interesting is the fact that the asteroid belt appears to be populated by small spaceships, similar in appearance to the Ka'het that infest the nearby systems but at the same time unlike any of the types you had seen before. Perhaps the Remnant can tell you more about this system and its sudden appearance.`
+		fail
+	to complete
+		never
+
+mission "Ka'het: Patir Mystery 1"
+	landing
+	name "Visit <waypoints>"
+	description "Bring Taely with you to visit the <waypoints> system before going to <destination>."
+	source
+		government "Remnant"
+	destination "Aventine"
+	waypoint "Patir"
+	passengers 1
+	cargo "Ka'het tracking equipment" 5
+	to offer
+		has "Ka'het: Patir Mystery 0: failed"
+		not "dusk busy"
+		random < 50
+	on offer
+		"dusk busy"++
+		conversation
+			`You start looking for Taely right after finishing the post-landing procedures. You aren't surprised to find her in the local shipyard, overseeing the welding of some kind of unmanned satellite. Though not familiar enough with Remnant technology to understand its function, you identify a large crystal placed near what you presume to be the front of the probe as a Quantum Key Stone, or at least a derivative of it.`
+			`	When Taely notices you walking towards her, she waves at you. "<first>! It has been a while. Were you looking for me?"`
+			choice
+				`	"I have some news. A new black hole suddenly appeared near the Graveyard, the same way Ssil Vida did."`
```

Comment:
I think this might revolve around the nuance of how the characters think about things. Do our NPCs primarily think about systems, and stars and other stellar features are things that are found inside systems? Or do they think of systems as regions of space that form around stellar features?

I, personally, think of them in the latter way. Systems are not things that just happen in the void - they are spaces defined by the star (or black hole, or other notable feature) that they encircle.

In this particular case, the terminology has a subtle meaning: Was the black hole always there, and suddenly there is a system around it which enables us to visit it? Or did the black hole just appear, bringing a system into existence alongside it?

My understanding of the story is that there was nothing there, and now there is a black hole, and the appearance of the black hole has also brought into existence the system.